aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -31,31 +31,40 @@ static inline Tensor _fft(const Tensor &self, const int64_t signal_ndim,
   }
   if (!at::isFloatingType(self.type().scalarType())) {
     std::ostringstream ss;
-    ss << "Expected an input tensor of floating types, but got input "
+    ss << "Expected an input tensor of floating types, but got input="
        << self.type() << self.sizes();
     throw std::runtime_error(ss.str());
```

Comment:
Shouldn't these be AT_ERROR now?

Reply:
Oh didn't know that. Let me fix those.

Reply:
Thanks!

Reply:
`AT_ERROR` doesn't support `std::string`, and I'm not sure if `c_str` is best practice. many other similar `std::runtime_error`s are not changed to `AT_ERROR` either. Since this is not the focus of this PR, I think I will leave them as is for now, and wait for a future codemod.